src/app/static/src/app/store/genericChart/actions.ts

```diff
@@ -29,15 +31,15 @@ export const actions: ActionTree<GenericChartState, DataExplorationState> & Meta
         await api<GenericChartMutation, GenericChartMutation>(context)
             .ignoreSuccess()
             .withError(GenericChartMutation.SetError)
-            .freezeResponse()
             .get(payload.url)
             .then((response) => {
                 if (response) {
+                    const frozenData = freezer.deepFreeze(response.data);
```

Comment:
I think it would be preferable to do this in the api client - would expect that calling `freezeResponse` means both that auto-comitted payloads are frozen but also that the returned `response` object is frozen (and fixing that would prevent this being missed again elsewhere.)